### ISSUE TITLE
Power? (hp, ft-lb)

### ISSUE THREAD

**Issue by keli5** [convert-units/convert-units#157]
It would be nice if horsepower and foot-pounds were included - not sure where these would fit. Thank you!

**Comment by whizyrel** [comment]
It would fit in [here](https://github.com/Brenopms/convert-units/blob/367eec5449b4b51c13d14a295765f90dbbaf971e/lib/definitions/power.js#L39) for Horsepower. Something like this, I think:
```json
"HP": {
    "name": {
      "singular": "Horsepower",
      "plural": "Horsepowers"
    },
   "to_anchor": 745.7
}
```
I would really appreciate it if you can implement this and go through with a PR @keli5. Thank you
